addOns/help/src/main/javahelp/contents/credits.html

```diff
@@ -234,9 +234,9 @@ <H2>3rd Party Libraries and Files</H2>
 <tr><td>&nbsp;&nbsp;&nbsp;&nbsp;</td><td>
 JavaHelp</td><td> <a href="http://java.sun.com/javase/technologies/desktop/javahelp/">http://java.sun.com/javase/technologies/desktop/javahelp/</a></td></tr>
 <tr><td>&nbsp;&nbsp;&nbsp;&nbsp;</td><td>
-OWASP DirBuster</td><td> <a href="https://www.owasp.org/index.php/Category:OWASP_DirBuster_Project">https://www.owasp.org/index.php/Category:OWASP_DirBuster_Project</a></td></tr>
```

Comment:
https://sourceforge.net/projects/dirbuster/

??

Reply:
Fine linking to OWASP wiki pages, they seem to have more information.

The text of the link needs to be updated.

Reply:
`Historic OWASP DirBuster content`?
`Archived OWASP DirBuster content`?

Reply:
Last one maybe.

Reply:
The text of the hyperlink is still saying `www.` though.

Reply:
Oh oops, I totally took your comment the wrong way. I'll tweak those too.